pkg/client/buildconfigs.go

```diff
@@ -73,7 +74,7 @@ func (c *buildConfigs) Get(name string) (result *buildapi.BuildConfig, err error
 func (c *buildConfigs) WebHookURL(name string, trigger *buildapi.BuildTriggerPolicy) (*url.URL, error) {
 	switch {
 	case trigger.GenericWebHook != nil:
-		return c.r.Get().Namespace(c.ns).Resource("buildConfigs").Name(name).SubResource("webhooks").Suffix(trigger.GenericWebHook.Secret, "generic").URL(), nil
+		return c.r.Get().Namespace(c.ns).Resource("buildConfigs").Name(name).SubResource("webhooks").Suffix(trigger.GenericWebHook.Secret, "generic", strconv.FormatBool(trigger.GenericWebHook.AllowEnv)).URL(), nil
```

Comment:
no, i just meant when printing out the describe output, you should show something like:
```
Triggered by:		Config
Webhook GitHub:
  URL: https://localhost:8443/oapi/v1/namespaces/p2/buildconfigs/nodejs-ex/webhooks/WJBHcA71E88k0NG508pA/github
Webhook Generic:	
  URL: https://localhost:8443/oapi/v1/namespaces/p2/buildconfigs/nodejs-ex/webhooks/XxAqfARQg26R6LUrwSgj/generic
  AllowEnv: true
```